x/superfluid/keeper/epoch.go

```diff
@@ -33,7 +33,7 @@ func (k Keeper) AfterEpochStartBeginBlock(ctx sdk.Context) {
 
 	ctx.Logger().Info("Distribute Superfluid gauges")
 	//nolint:errcheck
-	osmoutils.UnmeteredApplyFuncIfNoError(ctx, func(cacheCtx sdk.Context) error {
+	osmoutils.ApplyFuncIfNoError(ctx, func(cacheCtx sdk.Context) error {
```

Comment:
For this and the ones below, can you change ApplyFuncIfNoError to do the same as the one in hooks.go to print an error log so it's not failing completely silently?

Reply:
i don't get it. ApplyFuncIfNoError does this for you
```
ctx.Logger().Error(err.Error())
```

Reply:
Oh its double logged in hooks, wiht epoch as context. Imo this is fine, but if you'd like we can. This is only doing something for panics which contains the stack trace anyway.

Reply:
Ah I didn't see the log line in the method itself, seems fine as is, was using the previous call in hooks as the example